test/src/main/java/org/apache/accumulo/test/functional/ManagerApiIT.java

```diff
@@ -60,12 +60,14 @@ public class ManagerApiIT extends SharedMiniClusterBase {
   private static Credentials rootUser;
   private static Credentials regularUser;
   private static Credentials privilegedUser;
+  private static InstanceId instanceId;
 
   @Override
   protected Duration defaultTimeout() {
     return Duration.ofMinutes(1);
   }
 
+  @SuppressWarnings("deprecation")
```

Comment:
Instead of suppressing the entire method, can this be narrowed down to the specific deprecated item?

Reply:
Tried to, wasn't able to figure it out.